src/components/PostTypeChoiceModal/PostTypeChoiceModal.tsx

```diff
@@ -41,7 +45,13 @@ const PostTypeChoiceModal = () => {
   const onChooseType = (type: string) => {
     if (targetId && targetName && targetType) {
       const targetscreen =
-        type === 'post' ? 'CreatePost' : type === 'poll' ? 'CreatePoll' : null;
+        type === 'post'
+          ? 'CreatePost'
+          : type === 'poll'
+          ? 'CreatePoll'
+          : type === 'livestream'
+          ? 'CreateLivestream'
+          : null;
```

Comment:
Optional, In my opinion if it is a dedicate function which handle as early return for each type it should easier to read